src/render.ts

```diff
@@ -140,6 +140,12 @@ export function codeBlock(
  * @group render_utils
  */
 export function table(table: { rows: string[][]; columns: string[] }): string {
+  const columnLength = table.columns.length;
+  if (table.rows.some((row) => row.length !== columnLength)) {
```

Comment:
Some rows might have empty leading values in their cells so this can happen

Reply:
ok, you are right